src/Framework/BuildCheck/BuildCheckEventArgs.cs

```diff
@@ -28,6 +28,12 @@ internal sealed class BuildCheckTracingEventArgs(Dictionary<string, TimeSpan> tr
     internal BuildCheckTracingEventArgs() : this(new Dictionary<string, TimeSpan>())
     { }
 
+    internal BuildCheckTracingEventArgs(Dictionary<string, TimeSpan> data, bool isLogReport) : this(data)
+    {
+        IsLogReport = isLogReport;
+    }
+
+    public bool IsLogReport { get; private set; } = false;
```

Comment:
Just wondering if we can have bit more descriptive naming - something like?

```suggestion
    public bool IsAggregatedGlobalReport { get; private set; } = false;
```

Plus a comment would be nice.